entrypoint

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+set -e
+set -u
+
+: "${UID:=0}"
+: "${GID:=${UID}}"
+
+if [ "$#" = 0 ]; then
+        set -- "$(command -v bash 2>/dev/null || command -v sh)" -l
```

Comment:
Why is this needed?

Reply:
This is needed to set the default command to execute. When the user don't provide any argument `"$#" = 0` this will set the default command to the full path bash with sh as fallback. The -l  makes it a login shell so it will read the   .bashrc, .bash_profile, etc.

So this will sets the positional parameters to `/usr/bin/bash -l` that replacing the current shell an the end `exec "$@"`